src/renderer/main.js

```diff
@@ -1,10 +1,13 @@
 import Vue from 'vue'
+import VueCarousel from 'vue-carousel'
 
 import App from './App'
 
 if (!process.env.IS_WEB) Vue.use(require('vue-electron'))
 Vue.config.productionTip = false
 
+Vue.use(VueCarousel)
```

Comment:
By doing this, you register `VueCarousel` components globally (which is bad for bundle size in my opinion). I think you can just remove this line and keep importing `VueCarousel`components locally like you did in `App.vue`

Reply:
Do you have some link about the bundle part ?

Reply:
I'm not sure about this because as you import and use it somewhere, it will be include into your bundle. If you do component lazy loading, it could be useful but with a Eletron project not sure 😉   

Another concern I have about this, if you use `Vue.use()` into every component which need it, the install method will only be called for the `Vue.use()`. But what happened if you use some options: 
```
// component1.vue
Vue.use(Plugin, { option1: true }

// component2.vue
Vue.use(Plugin, { option1: false }

// component3.vue
Vue.use(Plugin, { anotherOption: 'Vue roxx' }
```

Which options will be apply as the registration is global ? I think it can be really thoublesome to debug 😕

Reply:
The only thing I can give you is [vuejs plugin doc](https://fr.vuejs.org/v2/guide/plugins.html). You can also see `vue-caroussel` `Global` and `Local` sections [here](https://github.com/SSENSE/vue-carousel).  
Basically if you directly import `vue-carousel` plugin in your app (`Vue.use(VueCarousel)`), you make all `vue-carousel` components (and other stuff from the plugin) available in each components of your app (In this case you don't need to import locally `vue-carousel` components in your own components) so your bundle size might be bigger.  

If you use local imports instead (like you did in `App.vue`) you will just take what you need from the plugin, so your bundle size might be smaller.  

I don't know if `vue-carousel` plugin expose more than these two components... If so, we could just use `Vue.use(VueCarousel)` 😄

Reply:
I read the `vue-carousel` documentation and it doesn't seems to be more than two components in the plugin, so bundle size remains the same no matter which method you choose. But you can still remove local imports from your `App.vue` 😃

Reply:
@Taranys you're right I forgot it wasn't a web project 😄

Reply:
Ok, I understand better. I thought I had to use `Vue.use()` inside every component which use `vue-carousel`...

The local import is much better 💟